pkg/apis/eventing/v1alpha1/broker_types.go

```diff
@@ -70,19 +70,7 @@ var brokerCondSet = duckv1alpha1.NewLivingConditionSet(BrokerConditionIngress, B
 
 // BrokerStatus represents the current state of a Broker.
 type BrokerStatus struct {
-	// ObservedGeneration is the most recent generation observed for this Broker.
-	// It corresponds to the Broker's generation, which is updated on mutation by
-	// the API Server.
-	// TODO: The above comment is only true once
-	// https://github.com/kubernetes/kubernetes/issues/58778 is fixed.
-	// +optional
-	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
-
-	// Represents the latest available observations of a broker's current state.
-	// +optional
-	// +patchMergeKey=type
-	// +patchStrategy=merge
-	Conditions duckv1alpha1.Conditions `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	duckv1alpha1.Status `json:",inline"`
```

Comment:
This is a net negative in terms of understanding what fields are actually in the status object when looking at the code.

It's slightly better in the generated docs because I can click the `Status` link to get the docs for the duck Status object (though it isn't documented as clearly as it is here): 
https://godoc.org/github.com/knative/eventing/pkg/apis/eventing/v1alpha1#ChannelStatus
https://godoc.org/github.com/knative/serving/pkg/apis/serving/v1alpha1#ConfigurationStatus

If we're going to use embedded structs here, those embeds need to be documented just as clearly as the individual fields were.

Reply:
https://github.com/knative/serving/pull/3444/files#diff-550eee22698eaa7e95e111aaf59f330bR259

Serving did not.

Reply:
Serving maintainers can do whatever they want, but in eventing we want our API objects to be documented 😎

Reply:
Clicking through to https://godoc.org/github.com/knative/pkg/apis/duck/v1alpha1#Status does not seem so bad to me